addon/services/unified-event-handler.js

```diff
@@ -106,7 +106,7 @@ export default Ember.Service.extend(Ember.Evented, {
     if (!handlerInfo) {
       // Add new DOM event listener since there is none
       let emberEventName = `${eventName}.${generateId()}`;
-      let trigger = this.triggerEvent.bind(this, emberEventName);
+      let trigger = this.triggerEvent.bind(this, target, eventName);
```

Comment:
Why is this change needed? Also, It seems like `emberEventName` isn't used here any more.

Reply:
This change is needed because I need `eventName` to lookup `handlerInfo` later on. FYI, `eventName` would be `scroll`, whereas `emerEventName` would be `scroll.1`, or `scroll.2`, etc.